installer/client/src/pages/WelcomeScreen.vue

```diff
@@ -108,6 +108,10 @@ export default {
   },
   methods: {
     toggleModal() {
+      if (this.selected === 'install') {
+        navigate('/../installerUI.php');
+        return;
+      }
```

Comment:
not sure requirement. cz `installerUI.php` legacy installer already removed

Reply:
then how to redirect to  the installation page. when we click the fresh installation